pkg/plugins/github/pullrequest.go

```diff
@@ -10,21 +10,19 @@ import (
 )
 
 // PULLREQUESTBODY is the pull request template used as pull request description
-const PULLREQUESTBODY string = `
-
-## Changelog
-
-{{ .Description }}
-
-## Reports
-
-{{ .Report }}
-
-## Remark
-
-This pull request was automatically created using [Updatecli](https://www.updatecli.io).
-Please report any issues with this tool [here](https://github.com/updatecli/updatecli/issues/new)
-`
+const PULLREQUESTBODY = "## Report\n" +
+	"{{ .Report }}" +
+	"\n" +
+	"## Changelog\n" +
+	"<details><summary>Click to expand</summary>\n" +
+	"\n```\n" +
+	"{{ .Description }}\n" +
+	"```\n" +
+	"</details>\n" +
+	"\n" +
+	"## Remark\n" +
+	"This pull request was automatically created using [Updatecli](https://www.updatecli.io).\n" +
+	"Please report any issues with this tool [here](https://github.com/updatecli/updatecli/issues/new)\n"
```

Comment:
```suggestion
// Please note that triple backticks are concatenated with the literals, as they cannot be escaped
const PULLREQUESTBODY = `

## Report

{{ .Report }}

## Changelog

<details><summary>Click to expand</summary>

` + "```\n{{ .Description }}\n```" + `

</details>

## Remark

This pull request was automatically created using [Updatecli](https://www.updatecli.io).

Please report any issues with this tool [here](https://github.com/updatecli/updatecli/issues/new)

`
```

WDYT about trying to keep the string litteral as much as possible, to improve lisibility (the rationale is that markdown requires a LOT of line returns and using the `\n` is painful to read)

Reply:
I hesitated to mix the syntax but I agree that it's more lisible